src/index.scss

```diff
@@ -37,11 +35,26 @@ body {
       li {
         .link {
           text-decoration-line: none;
+          color: black;
         }
       }
     }
   }
 
+  .svg-user {
+    color: #3d78ee;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.main-container {
+  background-color: inherit;
+  width: 70%;
+  margin: 20px auto;
+  // box-shadow: 1px 2px 25px rgb(131, 133, 130);
+  min-height: 600px;
+
```

Comment:
- [ ] Just as the previous reviewer mentioned make it look like the origin template, 
- [ ] which already your are doing 👏 👏 
- [ ] however please increase the  width `title` `author` and `category` then move it to right like the template 
- [ ] also kindly bookstore heading starts where the first book is starting from.
- [ ] Please see screenshot and highlighted areas for easy identification 
![image](https://user-images.githubusercontent.com/66230306/201497085-3adb221e-1998-4498-8a3d-ec82b39185e9.png)

Reply:
Okay, thanks. I will do it know.